src/vs/workbench/electron-browser/desktop.main.ts

```diff
@@ -122,6 +124,12 @@ class DesktopMain extends Disposable {
 		// Startup
 		const instantiationService = workbench.startup();
 
+		// Lifecycle Listeners
```

Comment:
If we move `DesktopLogService` to `src/vs/workbench/services/log/electron-browser` we could actually simply contribute a workbench contribution from there as well that starts on that phase. Would make it nicely selfcontained.